### PR TITLE
fix: made visibility icon responsive to small resolutions

### PR DESCRIPTION
**Pull request by vickywane**
This resolves: #513 

This PR contains fixes made to make the `peerID Visibility Icon` responsive to smaller resolutions.
The applied solution is to make proper use of `media queries` for the input container and also divide the PeerID's length by 2 when hidden. This ensures it doesn't overflow the appropriate space.

**Note:** The `peerID` length is only divided when hidden. The full value is revealed when visible. 

See attached screenshots:

media-queries

### CODE REVIEW

**Comment by vickywane** [comment]
For some unexplainable reason, the lighthouse check is exiting with an `Error: Timeout reached: Unable to connect to https://deploy-preview-514--ambianic-pwa-dist.netlify.app` error after logging `URL unavailable`. 

Is there a way to re-trigger the check with pushing again ( I don't have any new thing to push).

**Comment by ivelin** [comment]
@vickywane Lighthouse PWA CI / build check fails.

**Review by ivelin — CHANGES_REQUESTED** [review]
This works at the moment, but the value dividing doesn't feel right. The size of the text field should be controlled by the screen size and not by the length of the text is contains.

What if sometime down the road we have to switch to public keys for peer ID value, which can be 500+ characters long?

Please look closer at the vuetify responsive attributes that ensure components fit well. The solution you found for the buttons going outside mobile screen boundaries was great.

Let's avoid custom responsive code with hardcoded pixel values that may conflict with the vuetify built-in component layout algorithm.

See vuetify grids: https://vuetifyjs.com/en/components/grids/

**Review by ivelin — CHANGES_REQUESTED** [review]
One issue and one question:

1. The left alignment of the ID text seems off on screens bigger than iphone 5. Have you tested responsiveness thoroughly before requesting PR review? See screenshots.
2. The placement of the eye icon is space efficient. I like that. However it does not make it obvious that it's an interactive button as opposed to a static icon like the rest of the icons in that column of the screen. The user has to guess which icon does what. May be confused that all icons but one  have no action associated with them. How do we address that with some sort of visual cue that distinguishes a button icon from a static one?

Screen Shot 2021-02-06 at 5 48 30 PM
Screen Shot 2021-02-06 at 5 48 37 PM
Screen Shot 2021-02-06 at 5 48 42 PM
Screen Shot 2021-02-06 at 5 48 52 PM
Screen Shot 2021-02-06 at 5 48 59 PM
Screen Shot 2021-02-06 at 5 49 05 PM

**Comment by ivelin** [comment]
@vickywane lets add responsiveness tests that validate component visibility and layout under various device modalities: mobile (top 10 popular models), tablet, desktop:

Resources:
- https://maxschmitt.me/posts/cypress-testing-mobile-tablet-desktop/
- https://docs.cypress.io/api/commands/viewport.html#Usage
- https://medium.com/flux-it-thoughts/responsive-testing-cypress-72d2be68690b
- https://docs.cypress.io/guides/tooling/visual-testing.html#Functional-vs-visual-testing
- https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Visibility

File: src/views/Onboarding.vue [hunk 1]
```diff
@@ -694,7 +694,7 @@ export default {
         navigator.share({
           title: 'Ambianic Edge Device Access Request',
           text:
-            'Hi Blob, please send me an access invitation to your Ambianic Edge Device'
+              'Hi Blob, please send me an access invitation to your Ambianic Edge Device'
```
Comment (ivelin) [inline src/views/Onboarding.vue:697]: @vickywane  Please change "Blob" to "_____".  Blob may be offensive to some users.

File: src/components/shared/ListItem.vue [hunk 2]
```diff
@@ -93,6 +96,13 @@ export default {
       default: null
     }
   },
+  methods: {
+    divideIdValue (id) {
+      const arr = id.split('')
+      arr.splice(0, arr.length / 2)
```
Comment (ivelin) [inline src/components/shared/ListItem.vue:102]: @vickywane Dividing the text length by two is not an optimal solution. The value can be much longer than it is now in the future (e.g. a 500 char public key ), but that should not break the page. 

The good news however is that due to your latest use vuetify layout markup, I think you can just get rid of this `divideIdValue` function and everything should still look fine.

**Review by ivelin — CHANGES_REQUESTED** [review]

File: cypress/integration/ambianic-tests/remote-connection.spec.js [hunk 3]
```diff
@@ -28,4 +34,22 @@ context('RemoteConnections', () => {
         cy.get('#toggle-visibility').click()
         cy.get('#peerId-container').should('have.value', '917d5f0a-6469-4d33-b5c2-efd858118b74')
     })
-})
\ No newline at end of file
+
+    it('Displays elements in smaller viewports', () => {
```
[the remark this answers is not in the document]
Comment (vickywane) [reply cypress/integration/ambianic-tests/remote-connection.spec.js:38]: Sure, this would be taken note of.

File: src/views/Onboarding.vue (hunk 1 above)
Comment (ivelin) [inline src/views/Onboarding.vue:697]: The same request text appears twice in the same file. One is now using ___ and the other Blob....Is there a better way to handle repetitive literals?

**Review by ivelin — CHANGES_REQUESTED** [review]

**Pull request merged**